examples/admin_create_topic/admin_create_topic.go

```diff
@@ -20,7 +20,7 @@ package main
 import (
 	"context"
 	"fmt"
-	"github.com/confluentinc/confluent-kafka-go/kafka"
+	"gopkg.in/confluentinc/confluent-kafka-go.v1/kafka"
```

Comment:
This is a separate, and troublesome change.

Standalone applications should use the gopkg import, but the examples in the repo needs to use the github module url to make sure in-tree examples actually use the in-tree kafka/ package, and not a gopkg.in clone.